brayns/engine/SharedDataVolume.cpp

```diff
@@ -78,9 +78,15 @@ void SharedDataVolume::mapData(const std::string& filename)
     setVoxels(_memoryMapPtr);
 }
 
-void SharedDataVolume::mapData(const std::vector<char>& buffer)
+void SharedDataVolume::mapData(const uint8_ts& buffer)
 {
     _memoryBuffer.insert(_memoryBuffer.begin(), buffer.begin(), buffer.end());
     setVoxels(_memoryBuffer.data());
 }
+
+void SharedDataVolume::mapData(uint8_ts&& buffer)
+{
+    _memoryBuffer = std::move(buffer);
```

Comment:
Do you need the move here? Is this not already an rvalue reference?

Reply:
Yes you need it, buffer is an rvalue reference in the declaration, but once inside the function body it becomes an lvalue because it's named and it can appear at the left side of an assignment.